entries/f/fragdenstaat.de.json

```
@@ -0,0 +1,13 @@
+{
+  "FragDenStaat": {
+    "domain": "fragdenstaat.de",
+    "tfa": [
+      "totp",
+      "u2f"
+    ],
+    "documentation": "https://fragdenstaat.de/hilfe/ihr-konto-verwalten/zwei-faktor-login/",
+    "keywords": [
+      "government"
+    ]
```

There may not be a restriction for the accounts, but given the target audience we should probably use a region.
```suggestion
    ],
    "regions": [
      "de"
    ]
```